site/docs/spark.md

```diff
@@ -347,6 +347,28 @@ df.createOrReplaceTempView("table")
 spark.sql("""select count(1) from table""").show()
 ```
 
+**Note**: Dataframe read options are available as static constants in [SparkReadOptions](https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/types/SparkReadOptions.java) class.
+
+| Spark Read option    | Constant               |
+| --------------- | --------------------- |
+| snapshot-id     | SNAPSHOT_ID              |
+| as-of-timestamp | AS_OF_TIMESTAMP              |
+| split-size      | SPLIT_SIZE |
+| lookback        | LOOKBACK |
+| file-open-cost  | FILE_OPEN_COST |
+| vectorization-enabled  | VECTORIZATION_ENABLED |
+| batch-size  | VECTORIZATION_BATCH_SIZE |
```

Comment:
I'm not sure that we want to encourage users to refer to constants in Iceberg. What do you think, @aokolnychyi, @holdenk, @RussellSpitzer?